sources/engine/Stride.Rendering/Rendering/MeshDraw.cs

```diff
@@ -19,5 +22,28 @@ public class MeshDraw
         public VertexBufferBinding[] VertexBuffers;
 
         public IndexBufferBinding IndexBuffer;
+
+        public List<Vector3> VCLIST = new List<Vector3>();
+
+        public List<Tuple<int, int, Vector3>> VCPOLYIN = new List<Tuple<int, int, Vector3>>();
```

Comment:
Not too sure what those two fields are about, can you think of a name that would be more descriptive for those two ?